packages/ui/.storybook/main.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/preset-create-react-app",
+    "@storybook/addon-controls",
```

Comment:
What would that help us with?

Reply:
Deleted already too. I added to experiment because I thought we may need it. I should have asked first since I don't know how to really use it.

Reply:
Well we might wanna use it but as you say we need to investigate if it is of any use for us first

Reply:
I reckon this is not deleted tho, can you double check?

Reply:
@AdrianValdes check this comment out please ☝️

Reply:
Oh, for something reason I was not able to see this comment here, weird.

Reply:
So should I delete every addon? I thought you refer just to addon-controls, my bad

Reply:
I will delete these ones then:
` "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@storybook/preset-create-react-app"
    `
    They were there by default.

Reply:
Nope wait. Lets delete the one is added on this PR only and we can make a separate issue and discuss there possible addons that make sense.

Well I am saying this because you want in general to have a PR making one thing. Now if one PR makes multiple things then it starts being confusing for the reviewer and in general it isnt the best practise to mix things. 

Imagine a scenario that you merge a PR that by mistake breaks the application. In other words in pushes a bug in the application. In a real world app we want basically to be able to revert the changes pushed by that branch and _only_ those changes. Now if are used to have a PR does multiple things in scenarios like the one I am explaining you will end up not knowing what part of the changes broke the PR. 

This is why we keep it short and focused on one thing

Reply:
That makes sense, so we just deleted ` "@storybook/addon-controls",` that was the one I removed already.

Reply:
The others are there by default, so we will probably need them I suppose. Once I know more about storybook I will be able to have an opinion :)

Reply:
Exactly right!

Reply:
Perfect, I will merge then!